aws-nimblestudio-studiocomponent/aws-nimblestudio-studiocomponent.json

```diff
@@ -1,41 +1,31 @@
 {
   "typeName": "AWS::NimbleStudio::StudioComponent",
-  "description": "Represents a studio component which connects a non-Nimble Studio resource in your account to your studio",
+  "description": "Represents a studio component that connects a non-Nimble Studio resource in your account to your studio",
   "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-nimblestudio",
   "definitions": {
     "ActiveDirectoryComputerAttribute": {
       "type": "object",
-      "description": "<p>An LDAP attribute of an Active Directory computer account, in the form of a name:value pair.</p>",
+      "description": "<p>An LDAP attribute of an Active Directory computer account, in the form of a name:value\n            pair.</p>",
```

Comment:
Should the descriptions contain HTML tags? This newline probably should not be here. Newlines are represented differently in HTML.

Reply:
This is an auto generated file and is not modified manually. Other [files](https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-nimblestudio/blob/main/aws-nimblestudio-launchprofile/aws-nimblestudio-launchprofile.json#L20) in the repository have this type of marking too.